src/server/rest/v1/service/RegistrationTokenService.ts

```diff
@@ -58,6 +58,15 @@ export default class RegistrationTokenService {
         user: req.user
       });
     }
+    // Check site is provided for site admins
+    if (Authorizations.isSiteAdmin(req.user) && !filteredRequest.siteAreaID) {
```

Comment:
Only if the ORG component is active ;-)

Reply:
it's for security reason right ? because the tab is not active if ORG not active this is why I didn't think about it

Reply:
No, If the Org is not active there will be no checks with the Site Admin role but the Registration Token is still visible for the Admin